packages/simcore-sdk/src/simcore_sdk/node_ports_common/r_clone.py

```diff
@@ -63,13 +63,14 @@ async def is_r_clone_available(r_clone_settings: Optional[RCloneSettings]) -> bo
 async def sync_local_to_s3(
     local_file_path: Path,
     r_clone_settings: RCloneSettings,
-    upload_file_link: URL,
+    upload_file_links: FileUploadSchema,
 ) -> None:
     """_summary_
 
     :raises e: RCloneFailedError
     """
-    s3_link = urllib.parse.unquote(f"{upload_file_link}")
+    assert len(upload_file_links.urls) == 1  # nosec
```

Comment:
this will never happen per construction and per unit testing in storage
see https://github.com/ITISFoundation/osparc-simcore/blob/381187d5f8925c9976cb40057d8597884249db1a/services/storage/tests/unit/test_handlers_files.py
```python
pytest.param(
            MultiPartParam(
                link_type=LinkType.S3,
                file_size=parse_obj_as(ByteSize, "255GiB"),
                expected_response=web.HTTPOk,
                expected_num_links=1,
                expected_chunk_size=parse_obj_as(ByteSize, "255GiB"),
            ),
            id="5TiB file,s3",
        ),
    ],
)
async def test_create_upload_file_presigned_with_file_size_returns_multipart_links_if_bigger_than_99MiB(
```

Reply:
the assertion is more like an information to the developer